### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by RaoAkif**
Added the menu-bar popup:
- Added the JavaScript functionality to the navbar-icon
- Implemented the Close Navbar link

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -10,6 +25,68 @@ body {
...
+  }
+
+  .navbar-list-mobile li {
+    margin: 20px 0;
+  }
+
+  .navbar-list-mobile li a {
+    color: black;
+    text-decoration: none;
+  }
```
Comment (elmar8287) [inline style.css:74]: - [ ] Please, consider giving the mobile menu a design closer to the Figma template. Here are settings from the Figma:
```
font-family: Inter;
font-style: normal;
font-weight: 600;
font-size: 32px;
line-height: 44px;
display: flex;
align-items: center;
color: #3A4A42;
```
If you will use these styles for your `.navbar-list-mobile li a `, the mobile menu will look similar to the Figma template. 
> Your current style
Screen Shot 2022-03-02 at 00 41 53
:

> Figma style
Screen Shot 2022-03-02 at 00 42 03

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, team.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:

- Your mobile menu works correctly 👍 

## Required Changes ♻️

- [ ] Check the comments under the review.
- [ ] Please, consider adding JS linter. You can find by this [link](https://github.com/microverseinc/linters-config/tree/master/javascript), how to set up [ESLint](https://eslint.org/).


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (RaoAkif) [reply style.css:74]: Added the code. Now this is looking fine.

Comment (RaoAkif) [reply style.css:74]: Thanks for the suggestion.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @RaoAkif.

Great work on making the changes requested by me 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️

- [ ] Please, kindly fix your `linter.yml` file by adding the JS linter, it is required in the project. Please, go to this [link](https://github.com/elmar8287/leaderboard/blob/dev/.github/workflows/linters.yml) and copy/paste all content to your `linter.yml`. Please, after fixing, before pushing, check all linters for passing and then go on 👍 

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (elmar8287) [reply style.css]: > Added the code. Now this is looking fine.

Nice job - it looks perfect 👍 💯

**Pull request merged**